src/plugins/LADTable/plugin.js

```diff
@@ -19,38 +19,36 @@
  * this source code distribution or the Licensing information page available
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
+import LADTableViewProvider from './LADTableViewProvider';
+import LADTableSetViewProvider from './LADTableSetViewProvider';
+import LADTableCompositionPolicy from './LADTableCompositionPolicy';
 
-define([
-    './LADTableViewProvider',
-    './LADTableSetViewProvider'
-], function (
-    LADTableViewProvider,
-    LADTableSetViewProvider
-) {
-    return function plugin() {
-        return function install(openmct) {
-            openmct.objectViews.addProvider(new LADTableViewProvider(openmct));
-            openmct.objectViews.addProvider(new LADTableSetViewProvider(openmct));
+export default function plugin() {
+    return function install(openmct) {
 
-            openmct.types.addType('LadTable', {
-                name: "LAD Table",
-                creatable: true,
-                description: "A Latest Available Data tabular view in which each row displays the values for one or more contained telemetry objects.",
-                cssClass: 'icon-tabular-lad',
-                initialize(domainObject) {
-                    domainObject.composition = [];
-                }
-            });
+        openmct.objectViews.addProvider(new LADTableViewProvider(openmct));
+        openmct.objectViews.addProvider(new LADTableSetViewProvider(openmct));
 
-            openmct.types.addType('LadTableSet', {
-                name: "LAD Table Set",
-                creatable: true,
-                description: "A Latest Available Data tabular view in which each row displays the values for one or more contained telemetry objects.",
-                cssClass: 'icon-tabular-lad-set',
-                initialize(domainObject) {
-                    domainObject.composition = [];
-                }
-            });
-        };
+        openmct.types.addType('LadTable', {
+            name: "LAD Table",
+            creatable: true,
+            description: "A Latest Available Data tabular view in which each row displays the values for one or more contained telemetry objects.",
+            cssClass: 'icon-tabular-lad',
+            initialize(domainObject) {
+                domainObject.composition = [];
+            }
+        });
+
+        openmct.types.addType('LadTableSet', {
+            name: "LAD Table Set",
+            creatable: true,
+            description: "A Latest Available Data tabular view in which each row displays the values for one or more contained telemetry objects.",
+            cssClass: 'icon-tabular-lad-set',
+            initialize(domainObject) {
+                domainObject.composition = [];
+            }
+        });
+
+        openmct.composition.addPolicy(LADTableCompositionPolicy(openmct).allow);
```

Comment:
Ok, I see what's going on here now. You're following the pattern used in [ConditionSetCompositionPolicy](https://github.com/nasa/openmct/blob/master/src/plugins/condition/ConditionSetCompositionPolicy.js), which I've only just discovered.

`openmct.composition.addPolicy` simply accepts a function, not an object, so there's no need to instantiate any objects here. A simple alternative to creating an object here is to use function currying.

``` JavaScript
export default function ladTableCompositionPolicy(openmct) {
    return function (parent, child) {
        if(parent.type === 'LadTable') {
            return openmct.telemetry.isTelemetryObject(child);
        } else if(parent.type === 'LadTableSet') {
            return child.type === 'LadTable';
        }
        return true;
    }
}
```

Reply:
Updated! This makes sense, the other way was a bit overkill for the job.